paysage/models/state_.py

```diff
@@ -0,0 +1,13 @@
+# coding: utf-8
```

Comment:
This file shouldn't be needed.  We're only intending to support python3 in `paysage`.